edge/pkg/clusterd/edgecluster_state_reporter.go

```diff
@@ -81,16 +80,10 @@ func (esr *EdgeClusterStateReporter) initialEdgeCluster() (*edgeclustersv1.EdgeC
 
 func (esr *EdgeClusterStateReporter) prepareCluster() error {
 	basedir, _ := filepath.Abs(filepath.Dir(os.Args[0]))
-	missionCrdFilePath := filepath.Join(basedir, MissionCrdFile)
-	deployMissioCrdCmd := fmt.Sprintf("%s apply --kubeconfig=%s -f %s ", config.Config.KubectlCli, config.Config.Kubeconfig, missionCrdFilePath)
-	if _, err := helper.ExecCommandToCluster(deployMissioCrdCmd); err != nil {
-		return fmt.Errorf("Failed to deploy mission crd: %v", err)
-	}
-
-	edgeClusterCrdFilePath := filepath.Join(basedir, EdgeClusterCrdFile)
```

Comment:
any impact of removing these code?

Reply:
all the crd yaml files are under one folder now. Now, instead of deployed crd yaml files one by one, we deploy the whole folder. That makes things simpler.